openedx-unsupported/registrar
# Flesh out YAML API documentation

@edx/masters-neem 

**How to test**
* `git checkout kdmccormick/api-docs`
* `docker restart edx.devstack.registrar`
* Navigate to http://localhost:18734/login, log in as edx@example.com so all endpoints are visible
* Navigate to http://localhost:18734/api-docs and make sure the look right and are easy to understand
